core/src/main/java/tc/oc/pgm/score/MercyRule.java

```diff
@@ -0,0 +1,100 @@
+package tc.oc.pgm.score;
+
+import java.util.AbstractMap;
+import java.util.Map;
+import tc.oc.pgm.api.party.Competitor;
+import tc.oc.pgm.api.party.event.CompetitorScoreChangeEvent;
+
+public class MercyRule {
+
+  private final ScoreMatchModule scoreMatchModule;
+  private final int scoreLimit;
+  private final int mercyLimit;
+
+  private AbstractMap.SimpleEntry<Competitor, Double> leader;
+  private AbstractMap.SimpleEntry<Competitor, Double> trailer;
```

Comment:
```suggestion
  private Map.Entry<Competitor, Double> leader;
  private Map.Entry<Competitor, Double> trailer;
```
Just define the generric types here